ci/downstream-projects/func-spl.sh

```diff
@@ -19,7 +19,7 @@ spl() {
     )
     set -x
     rm -rf spl
-    git clone https://github.com/solana-labs/solana-program-library.git spl
+    git clone https://github.com/alessandrod/solana-program-library.git spl
```

Comment:
Should this be reverted back to `solana-labs`?

Reply:
@pgarg66 There are two commits at the end named "REVERT" these have to be reverted, so yes.